tests/module/mobject/mobject/test_mobject.py

```diff
@@ -29,12 +29,20 @@ def test_mobject_add():
     obj = Mobject()
 
     # a Mobject cannot contain itself
-    with pytest.raises(ValueError):
-        obj.add(obj)
+    with pytest.raises(ValueError) as add_self_info:
+        obj.add(Mobject(), obj, Mobject())
+    print(str(add_self_info.value))
```

Comment:
```suggestion
```
Is there a need to print it?

Reply:
Whoops, that shouldn't have been there. I just removed it.